system-test/storage.ts

```diff
@@ -2155,9 +2154,7 @@ describe('storage', () => {
             },
           });
         },
-        {
-          message: 'Cannot parse response as JSON: Metadata part is too large.',
-        }
+        (err: Error) => err.message.indexOf('Metadata part is too large.') > -1
```

Comment:
Should be able to simplify this entire line to:
```js
/Metadata part is too large/
```

That method takes an optional regex, or function to do the check.